eva-accession-release/src/main/java/uk/ac/ebi/eva/accession/release/steps/processors/ContextNucleotideAdditionProcessor.java

```diff
@@ -0,0 +1,70 @@
+/*
+ *
+ * Copyright 2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package uk.ac.ebi.eva.accession.release.steps.processors;
+
+import org.apache.commons.lang3.tuple.ImmutableTriple;
+import org.springframework.batch.item.ItemProcessor;
+
+import uk.ac.ebi.eva.accession.dbsnp.io.FastaSynonymSequenceReader;

```

Comment:
We already have a package `uk.ac.ebi.eva.accession.core.io.FastaSynonymSequenceReader` which contains the parent for this class. It would be preferable to move it there and have the core module declared in the POM file, instead of the import one.